packages/lesswrong/server/resolvers/commentResolvers.ts

```diff
@@ -105,4 +115,83 @@ augmentFieldsDict(Comments, {
       return (post && post.contents && post.contents.version) || "1.0.0";
     },
   },
+  doppelComments: {
+    resolveAs: {
+      type: '[DoppelComment]',
+      resolver: async (dbComment, args, context: ResolverContext) => {
+        const {DoppelComments, Users, currentUser} = context
+        if (!userHasDoppelComments(currentUser)) return []
+        // This should always be a single user
+        const [user] = await getWithLoader(context, Users, "usersByIdsForDoppelComments", {}, "_id", dbComment.userId, {karma: 1})
```

Comment:
`getWithLoader` was badly typed; in particular the `options` arg (the last one).  You should merge master and fix it if it's intended to be sorting by karma.  (But actually I don't really understand why this isn't just `context.loaders.Users.load(dbComment.userId)`.)